torch_geometric/loader/__init__.py

```diff
@@ -12,23 +12,13 @@
 from .temporal_dataloader import TemporalDataLoader
 from .neighbor_sampler import NeighborSampler
 from .imbalanced_sampler import ImbalancedSampler
+from .dynamic_batch_sampler import DynamicBatchSampler
 
 __all__ = classes = [
-    'DataLoader',
-    'NeighborLoader',
-    'LinkNeighborLoader',
-    'HGTLoader',
-    'ClusterData',
-    'ClusterLoader',
-    'GraphSAINTSampler',
-    'GraphSAINTNodeSampler',
-    'GraphSAINTEdgeSampler',
-    'GraphSAINTRandomWalkSampler',
-    'ShaDowKHopSampler',
-    'RandomNodeSampler',
-    'DataListLoader',
-    'DenseDataLoader',
-    'TemporalDataLoader',
-    'NeighborSampler',
-    'ImbalancedSampler',
+    'DataLoader', 'NeighborLoader', 'LinkNeighborLoader', 'HGTLoader',
+    'ClusterData', 'ClusterLoader', 'GraphSAINTSampler',
+    'GraphSAINTNodeSampler', 'GraphSAINTEdgeSampler',
+    'GraphSAINTRandomWalkSampler', 'ShaDowKHopSampler', 'RandomNodeSampler',
+    'DataListLoader', 'DenseDataLoader', 'TemporalDataLoader',
+    'NeighborSampler', 'ImbalancedSampler', 'DynamicBatchSampler'
```

Comment:
```suggestion
    'NeighborSampler', 'ImbalancedSampler', 'DynamicBatchSampler',
```
to let `yapf` re-format it.